### ISSUE TITLE
Using fine-tuned weights locally?

### ISSUE THREAD

**Issue by astralmedia** [facebookresearch/audiocraft#377]
Loving these developments...I recently have fine-tuned a model using replicate, and downloaded the weights...but cant seem to figure out how to use these locally...is there some step im missing? Doesnt seem like the fine tuned weights plug in.

Thank you for any feedback

**Comment by Frei-2** [comment]
@astralmedia 
Hello, recently I am trying to finetune musicgen but I find the docs in [MUSICGEN.md](https://github.com/facebookresearch/audiocraft/blob/main/docs/MUSICGEN.md) hard to follow for me and encounter a lot of errors when executing the finetuning command. I also download the pretrained model locally. 

I notice that you have tried to finetune a model of this project so I come here to seek some experience. 

Thanks in advance for any help and would appreciate any general tips for something else I might be doing wrong.
